### ISSUE TITLE
Slider field

### ISSUE THREAD

**Issue by danharrin** [laravel-filament/filament#12728]

**Comment by zepfietje** [comment]
Could you add a little more context here, @danharrin? 🙂

**Comment by danharrin** [comment]
This library was the original suggestion for implementation https://refreshless.com/nouislider/

**Comment by danharrin** [comment]
I'm happy to do the bulk of the work as long as you take a look at the design

**Comment by zepfietje** [comment]
Why not use the native HTML slider?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

**Comment by danharrin** [comment]
I don't think you can add multiple points onto that

**Comment by binaryfire** [comment]
@zepfietje Yeah the native slider has a fair few limitations vs noUi. noUi has multiple points, better step options, much smoother touch functionality etc.

Also a big plus is the events. You’d need to implement some custom debounce logic to avoid the same DDoS problem the color picker had (where updates were sent for every tiny movement). noUi has a bunch of events that could be tapped into for this.

**Comment by danharrin** [comment]
@binaryfire since you requested this, I thought I'd let you know. Polar is now active so you can donate to support us in fulfilling your feature.
